airbyte-cdk/java/airbyte-cdk/typing-deduping/src/testFixtures/java/io/airbyte/integrations/base/destination/typing_deduping/BaseSqlGeneratorIntegrationTest.java

```diff
@@ -423,7 +411,10 @@ public void allTypes() throws Exception {
         dumpRawTableRecords(streamId),
         "sqlgenerator/alltypes_expectedrecords_final.jsonl",
         dumpFinalTableRecords(streamId, ""));
-    assertFalse(destinationHandler.isFinalTableEmpty(streamId), "Final table should not be empty after T+D");
+    initialStates = destinationHandler.gatherInitialState(List.of(incrementalDedupStream));
```

Comment:
nit: maybe extract a getOnly function?

```java
private <T> T getOnly(List<T> list) {
  assertEquals(1, list.size(), "Expected list to have exactly one element. Got " + list.size() + " elements: " + list);
  return list.getFirst();
}
```

then you can change this block to something like
```java
initialStates = destinationHandler.gatherInitialState(List.of(incrementalDedupStream));
assertFalse(getOnly(initialStates).isFinalTableEmpty(), "Final table should not be empty after T+D");
```

Reply:
Changed. didn't use generic, just concrete type (only 2 uses)